package/AUTHORS

```diff
@@ -83,7 +83,7 @@ Chronological list of authors
   - Utkarsh Bansal
   - Shobhit Agarwal
   - Vedant Rathore
-  - Xiki Tempula
+  - Zhiyi Wu
```

Comment:
Oops... not sure who did that. Thanks for correcting.

Reply:
@orbeckst I did that last year actually, as most of my friends use the sort of fake name to remain anonymous when contributing to open-source projects. As I'm a graduate student now, I guess I should probably use the official name.